app/assets/javascripts/oxalis/view/right-menu/comment_tab/tree_comment_list.js

```diff
@@ -36,11 +35,19 @@ class TreeCommentList extends React.PureComponent<TreeCommentListProps, State> {
     this.ensureVisible();
   }
 
+  activeNodeHasComment(): boolean {
+    return this.props.tree.comments.reduce(
```

Comment:
Better `this.props.tree.comments.some(comment => comment.nodeId === this.props.skeletonTracing.activeNodeId);`